website/docs/teleop/leader-follower/setup-guide.md

```diff
@@ -27,13 +27,13 @@ cd openarm_teleop
 
 ```bash
 sudo apt update && sudo apt install -y \
-  libopenarmcan-dev \
```

Comment:
FYI: We'll provide this (in a few weeks): https://github.com/enactic/openarm_can/issues/33
We can update this part after that.